config/helix.properties

```diff
@@ -1,4 +1,4 @@
 zkServer=localhost:2181
 instanceId=testHelixMirrorMaker01
 helixClusterName=testMirrorMaker
-federated.deployment.name=uReplicator-Tests
+federated.deployment.name=testing-dc1
```

Comment:
please change it back to uReplicator-Tests?

Reply:
Done.